frontend/packages/client/src/components/Community/JoinCommunityButton.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from "react";
+import { useWebContext } from "contexts/Web3";
+import { useJoinCommunity, useUserRoleOnCommunity } from "hooks";
+
+export default function JoinCommunityButton({
+  communityId,
+  setTotalMembers = () => { },
+}) {
+  const { createCommunityUser, deleteUserFromCommunity } = useJoinCommunity();
+  const { injectedProvider, user } = useWebContext();
+  const { addr } = user;
+  const memberState = useUserRoleOnCommunity({
+    addr,
+    communityId,
+    roles: ["member"],
+  });
+  const [isMember, setMemberState] = useState(memberState);
```

Comment:
`isMember` is being passed as a prop to the component but here you are using a state variable with the same name. 

After investigating further it seems that `isMember` is always set to false and the same is happening for `enableJoin`.  Are you able to update the API endpoints to return this data or do we need Richard to help?

Reply:
Provided this information exists to be made use of, I can make those changes on Tuesday.

Reply:
@dbslone , this is done